docker/package/tezos_voting_wizard.py

```diff
@@ -257,7 +251,7 @@ def get_baker_key(self):
             self.config["baker_key_value"] = value
         else:  # if there is no key with this alias, query import
             key_mode_query = get_key_mode_query(key_import_modes)
-            self.import_key(key_mode_query)
+            self.import_key(key_mode_query, "Wallet")
 
     def get_network(self):
         if parsed_args.network == "mainnet":
```

Comment:
_leaving this comment here because GH doesn't allow me to leave it in a better place_

I noticed now the `Choose again?` option given when voting in the proposing period (aka submitting a proposal).

I think we should remove it, because I don't think it's that useful and if needed one can quickly re-launch the wizard and submit a new hash in a matter of seconds.

Reply:
Again, I disagree slightly :see_no_evil: In the proposal period, it's allowed to submit/vote up to 20 times. As the voting wizard must process these one-by-one and not in batches (note this is a conscious and IMO good decision, as it's [officially](https://obsidian.systems/blog/voting-on-tezos-with-your-ledger-nano-s) not recommended to batch-submit proposals when using ledgers), it's very likely that what you're proposing will lead to 20 invocations with a lot of "does this info seem right to you?" spam, i.e. it will actively bloat the amount of "clicks" (key presses, I guess) one needs to make in order to submit the maximum 20 proposals. The `Choose again?` question is a better alternative IMO.

Reply:
Oh, interesting, I didn't know about the limit of 20 proposals.

I agree that it's a better experience if one has to submit many of them, but ... how often will that happen.

AFAIU this only makes sense for _different_ proposal and from what I recall the most were proposed after (the first) ithaca was rejected at [period #64](https://agora.tezos.com/period/64) and it was only 4 for the whole community :shrug:

Reply:
Yeah, fair point. For some reason I thought duplicates counted as votes (e.g. 20 submissions for one proposal would boost its rate more than 1 submission), but I see that's not the case, so it definitely makes sense to exit right away.